resources/docs/introduction.md

```diff
@@ -20,6 +20,19 @@ Greetings {{ $friend }}, let's build Chirper with Blade!
     <span class="w-full inline-flex items-center justify-center self-stretch px-4 py-2 text-sm text-red-600 text-center font-bold uppercase bg-white dark:bg-dark-700 ring-1 ring-red-600 ring-offset-1 dark:ring-offset-dark-600 transform transition-transform group-hover:-translate-y-1 group-hover:-translate-x-1 group-focus:-translate-y-1 group-focus:-translate-x-1">Build Chirper with Blade</span>
 </a>
 
+### Livewire
+
+[Livewire](https://livewire.laravel.com/) is a powerful way of building dynamic, reactive, front-end UIs using just PHP. It's perfect for small teams that don't have the time to learn JavaScript, or for solo developers who'd rather write PHP than JS. If you're a Laravel developer, you'll feel right at home.
+
+```blade filename=welcome.blade.php
+Greetings {{ $friend }}, let's build Chirper with Livewire!
+```
```

Comment:
I wonder whether it's worth adding tabs for Class and Functional here, with a minimal representation of each, and then asking them up-front to choose an API as we do with Vue/React.

I love that both options are provided, but with the way they are presented later in the process without a code example of each, I didn't have any context to help me pick one.